src/test/unittest/devdax.py

```diff
@@ -18,17 +18,30 @@ class DevDax():
     """
     Class representing dax device and its parameters
     """
-    def __init__(self, name, alignment=None, min_size=None, max_size=None):
+    def __init__(self, name, alignment=None, min_size=None,
+                 max_size=None, deep_flush=None):
         self.name = name
         self.path = None
         self._req_alignment = alignment
         self._req_min_size = min_size
         self._req_max_size = max_size
+        self._req_deep_flush = deep_flush
         self.assigned = False
 
     def __str__(self):
         return self.name
 
+    def check_deep_flush_file(self, path):
+        """
+        Check if deep_flush file exists.
+        """
+
+        region_id = path.partition("/dev/dax")[2].partition(".")[0]
+        path_deep_flush = os.path.join("/sys/bus/nd/devices",
```

Comment:
Will this work on older kernels (<5.1)? Is this file affected by the change of `/sys/class/dax`?